File: src/org/cappuccino_project/ide/intellij/plugin/references/ObjJClassDeclarationProcessor.java
package org.cappuccino_project.ide.intellij.plugin.references;

import com.intellij.psi.search.PsiElementProcessor;
import org.cappuccino_project.ide.intellij.plugin.psi.interfaces.ObjJClassDeclarationElement;
import org.jetbrains.annotations.NotNull;

public class ObjJClassDeclarationProcessor implements PsiElementProcessor<ObjJClassDeclarationElement> {
    @Override
    public boolean execute(
            @NotNull
                    ObjJClassDeclarationElement classDeclarationElement) {
        return false;
    }
}

File: src/org/cappuccino_project/ide/intellij/plugin/references/ObjJClassDeclarationProcessor.kt
package org.cappuccino_project.ide.intellij.plugin.references

import com.intellij.psi.search.PsiElementProcessor
import org.cappuccino_project.ide.intellij.plugin.psi.interfaces.ObjJClassDeclarationElement

class ObjJClassDeclarationProcessor : PsiElementProcessor<ObjJClassDeclarationElement<*>> {
    override fun execute(
            classDeclarationElement: ObjJClassDeclarationElement<*>): Boolean {
        return false
    }
}
